src/python/WMCore/WebTools/RESTModel.py

```diff
@@ -185,6 +185,14 @@ def wrapper(*input_args, **input_kwargs):
             if len(args) != 0:
                 # store the method name
                 method = methodKey
+                # for PUT/POST requests data may be send in body of the request
+                # we check if this is the case (no args/kwargs) and read body
+                if  verb in ['PUT', 'POST'] and \
+                    not input_args and not input_kwargs:
+                    from WMCore.Wrappers import JsonWrapper
+                    body = cherrypy.request.body.read()
+                    if  body:
+                        input_kwargs = JsonWrapper.loads( body )
```

Comment:
Although I am not comfortable changing the RESTModel code since we don't have full control apis using this code.
I think relaxing the restriction on _sanitise_input will be OK.
I am still getting errors with this patch though with suggested change. Error messages are reported #6360